### DISCUSSION TITLE
Use Member Readme to Link Member Resources (if any)

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#36] [General]
**Was not to be a poll**

If we had member-only resources this would be great.

## Context

So, GitHub has a public and member-only README files for different profile contexts.

If you are a member you probably noticed this "Profile Context" switch:

![Public and Member Contexts](https://user-images.githubusercontent.com/108250871/192125316-160fd9b3-91df-457b-8562-aea7a9186812.png)

This switch will activate the different readmes that are available (separately, the "Public" and "Member" switches).

If we had member resources, this would be useful. We may set plans for this.

**Comment by confused-Techie** [comment]
Didn't realize we could have different readme's. But what do we think would be appropriate to add here that isn't available elsewhere?

**Reply by ghost** [reply]
I said, member-only resources. What I have in mind is inspired by the Primer design system by GitHub, which is a "releasing" file that contains docs for releasing versions. (The docs are public by GitHub for some reason)

**Reply by ghost** [reply]
We could then put them in a repository (I'd like it to be named `member-only-resources`).

**Reply by confused-Techie** [reply]
Unfortunately, if we do make this change, the repo the data is located in, looks like it has to be named `.github-private` from what I'm seeing on the [documentation](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)

**Reply by ghost** [reply]
> Unfortunately, if we do make this change, the repo the data is located in, looks like it has to be named `.github-private` from what I'm seeing on the [documentation](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)

That would be just the readme file. The resources can be linked straightforwardly to `member-only-resources`. (I know already about the .github-private thing, but thanks for reminding me)

**Reply by confused-Techie** [reply]
But okay, yeah we could have some versioning information in there. And I realize you said member only resources, I was just asking what those would be, since everything else is pretty open. Also maybe pointing out people to specifically ask about for items could be helpful

**Reply by ghost** [reply]
Also rn due to the permissions I could create a blank private `member-only-resources` repository that will be filled up once poll says yes.

**Reply by ghost** [reply]
Sidenote: When creating the stub private repository I found out that I can manage every part of the repository
![image](https://user-images.githubusercontent.com/108250871/192125682-6b2dd62f-2d82-479b-a4ec-a933fea7b57e.png)

**Reply by ghost** [reply]
> Also maybe pointing out people to specifically ask about for items could be helpful

What do you **actually** have for the *specifically ask about for items*

**Reply by ghost** [reply]
Also I have some basic permissions set up for each person: 
![image](https://user-images.githubusercontent.com/108250871/192126263-4dfb43cf-2f06-4190-8bc1-2ee92dcb5ddd.png)

**Reply by confused-Techie** [reply]
> Sidenote: When creating the stub private repository I found out that I can manage every part of the repository ![image](https://user-images.githubusercontent.com/108250871/192125682-6b2dd62f-2d82-479b-a4ec-a933fea7b57e.png)

That seems like a very obvious security flaw on GitHub's side. But thanks for pointing it out. Super interesting

**Reply by ghost** [reply]
> > Sidenote: When creating the stub private repository I found out that I can manage every part of the repository ![image](https://user-images.githubusercontent.com/108250871/192125682-6b2dd62f-2d82-479b-a4ec-a933fea7b57e.png)
> 
> That seems like a very obvious security flaw on GitHub's side. But thanks for pointing it out. Super interesting

That's probably obvious. Probably because they want people to have admin on their repositories created when they are simply members.

**Reply by confused-Techie** [reply]
> > Also maybe pointing out people to specifically ask about for items could be helpful
> 
> What do you **actually** have for the _specifically ask about for items_

Wait I'm sorry, I'm not sure I understand the question. Are you meaning do I have the resource already? Or are you meaning what would I put there

**Reply by ghost** [reply]
> > > Also maybe pointing out people to specifically ask about for items could be helpful
> > 
> > 
> > What do you **actually** have for the _specifically ask about for items_
> 
> Wait I'm sorry, I'm not sure I understand the question. Are you meaning do I have the resource already? Or are you meaning what would I put there

"What would I put there"

**Reply by ghost** [reply]
I'll delete the repository and recreate it once poll has led to final results

**Comment by meadowsys** [comment]
<img width="503" alt="image" src="https://user-images.githubusercontent.com/49562048/192127106-11a8445d-682e-46ee-828d-5cd705bc2568.png">

**Reply by ghost** [reply]
Umm... OK
